Templates/DefaultComponent/Template/Source/${Name}Component.cpp

```diff
@@ -0,0 +1,72 @@
+// {BEGIN_LICENSE}
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+// {END_LICENSE}
+
+#include <${SanitizedCppName}Component.h>
+
+#include <AzCore/Serialization/SerializeContext.h>
+#include <AzCore/Serialization/EditContext.h>
+#include <AzCore/RTTI/BehaviorContext.h>
+
+namespace ${GemName}
+{
+    void ${SanitizedCppName}Component::Activate()
+    {
+        ${SanitizedCppName}RequestBus::Handler::BusConnect(GetEntityId());
+    }
+
+    void ${SanitizedCppName}Component::Deactivate()
+    {
+        ${SanitizedCppName}RequestBus::Handler::BusDisconnect(GetEntityId());
+    }
+
+    void ${SanitizedCppName}Component::Reflect(AZ::ReflectContext* context)
+    {
+        if (AZ::SerializeContext* serializeContext = azrtti_cast<AZ::SerializeContext*>(context))
+        {
+            serializeContext->Class<${SanitizedCppName}Component, AZ::Component>()
+                ->Version(1)
+                ;
+
```

Comment:
Would it be useful to have it actually expose like, 1 field as an example?  Or hav ea comment here like
"When you add member fields to your components, reflect them like..."

Reply:
I'll do something like this when I add an example of using **component configuration** to the template in an update.